.github/workflows/labels-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Labels Check
+on:
+  pull_request:  
+    types: [opened, labeled, unlabeled, synchronize]
+    branches:
+      - main
+    paths-ignore:
+      - 'LICENSE'
+      - '**/.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
+      - '.github/**'
+      - 'kustomize/**'
+      - 'dev/**'
```

Comment:
I think this section must be aligned with [Manager CI job](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/Manager-CI.yaml#L7-L14) (please add a comment in both jobs to keep them in sync)